pipeline-modules/deployment-service/aws/lambda-gateway.yaml

```diff
@@ -0,0 +1,152 @@
+provisioner: aws
+name: lambda-gateway
+version: 1
+revision: 1
+displayName: "AWS Lambda deployment"
+description: "deploy lambda functions with codepipes and serverless"
+target: "deployment-template"
+keywords: ["bash", "linux","functions"]
+author: "CloudCover"
+meta:
+  inputArtifactType:
+    - ContainerImage
+inputs:
+  type: object
+  properties:
+    region:
+      title: Region name
+      description: region name for cluster
+      type: string
+    app_env:
+      title: Application Environment
+      description: Application Related Envs
+      type: object
+      default: {}
+    memory:
+      title: Memory
+      description: Memory to be consumed by app runner
+      type: string
+      default: "1024"
+    timeout:
+      title: timeout
+      description: timeout in seconds
+      type: integer
+      default: 10
+    project:
+      title: Project Name
+      description: This is a vanguard project name
+      type: string
+      default: latest
+    applicationName:
+      title: Application name
+      description: This is vanguard application name
+      type: string
+      default: latest
+    environment:
+      title: Environment Name
+      description: This is vanguard environment name
+      type: string
+      default: dev
+    architecture:
+      title: CPU architecture
+      description: This could be arm64 or amd_x64
+      type: string
+      default: arm64
```

Comment:
seems like the default should be amd, no?

Reply:
yes